src/ClientBuilder.php

```diff
@@ -64,6 +65,8 @@
  * @method setTags(string[] $tags)
  * @method bool shouldSendDefaultPii()
  * @method setSendDefaultPii(bool $enable)
+ * @method callable getBeforeSendCallback()
```

Comment:
This got out-of-sync to the `Options` definition, so I got unjustified PHPStan errors here.

IMHO we should ditch this `__call` magic, and use the `Options` object directly. As discussed elsewhere, I would like to pass the `Options` in the ctor, so the naked array requìred only with the `init()`. @ste93cry @HazAT WDYT?

Reply:
I'm +1 on this change, my intentions of making the builder a proxy for the options did not bring the desired results so let's remove everything and simplify things. I would just suggest to do this kind of change in a separate PR since it's not the main purpose of this one